pyo3-derive-backend/src/pyclass.rs

```diff
@@ -365,6 +380,9 @@ fn impl_class(
         #inventory_impl
 
         #extra
+
+        #gc_impl
```

Comment:
This macro will stop evaluating at the token `;` included from `gc_impl` for some reason. Similarly it will stop evaluating at `let` if you change this to `let x = 3`. You can move either of these statements around in the `quote!` macro to kill it earlier. Not sure why this is happening.